.circleci/config.yml

```diff
@@ -44,7 +44,6 @@ workflows:
       - test
       - test-build
       - test-integration-amd64
-      - test-integration-arm
```

Comment:
Add a todo to re-enable this. It shouldn't be broken

Reply:
it's reenabled :-)